protocol/avdl/keybase1/teams.avdl

```diff
@@ -877,4 +877,5 @@ protocol teams {
    */
   ProfileTeamLoadRes profileTeamLoad(LoadTeamArg arg);
 
+  TeamID getTeamIDByName(string teamName);
```

Comment:
I'd be happy to change this to `getTeamIDByNameUntrusted`.

The original plan was to have `trusted` (and maybe `public`) arguments to this function, but there's no way to set default values and it would make calling it unnecessarily complicated from JS side.

Reply:
I thought frontend usually wrapped everything. Who did you talk to about too many arguments?

Reply:
cc @buoyad

Reply:
Could be named `getTeamID`

Reply:
Comment that it has no caching and hits the server every time (if that's right).

Reply:
From a quick look at resolver it seems to be caching both in memory and on disk.

Reply:
I'll look again and watch logs to see when it makes requests though